package.json

```diff
@@ -14,10 +15,26 @@
     "Animation",
     "Physics"
   ],
-  "author": "Parker Ziegler",
+  "author": "Parker Ziegler <parker.ziegler@formidable.com>",
   "license": "MIT",
   "devDependencies": {
+    "@babel/core": "^7.7.4",
+    "@babel/preset-env": "^7.7.4",
+    "@glennsl/bs-jest": "^0.4.9",
+    "babel-jest": "^24.9.0",
     "bs-platform": "^6.2.1",
-    "gentype": "^3.6.0"
+    "gentype": "^3.6.0",
+    "jest": "^24.9.0"
+  },
+  "jest": {
+    "testMatch": [
+      "<rootDir>/__tests__/**/*_test.bs.js"
+    ],
+    "transform": {
+      "^.+\\.js$": "babel-jest"
+    },
+    "transformIgnorePatterns": [
+      "node_modules/(?!(@glennsl/bs-jest|bs-platform)/)"
+    ]
```

Comment:
This is a negative lookahead such that `babel-jest` **will** transpile these `node_modules`. Since our `bsconfig` specifies `es6-global` as the build target, we need to further transpile all BuckleScript output (ours and that of 3rd party libs) down to ES5. This is why we have this here.